setup.py

```diff
@@ -48,12 +48,13 @@
         # see conda/environment.yml
     ],
     setup_requires=[
-        # see environment.yml
+        # see conda/environment.yml
     ],
     tests_require=[
-        # see environment.yml
+        # see conda/environment.yml
     ],
     extras_require={
-        # see environment.yml
-    }
+        # see conda/environment.yml
+    },
+    package_data={'data': ['matchms/data/*.yaml']},
```

Comment:
According to [docs](https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files) it should be 
```suggestion
    package_data={'matchms': ['data/*.yaml']},
```

Also remove `include_package_data` entry, as it can conflict with `package_data`.